src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -1895,11 +1897,11 @@ RESULT_CODE ProtocolHandlerImpl::EncryptFrame(ProtocolFramePtr packet) {
         connection_key,
         security_manager::SecurityManager::ERROR_ENCRYPTION_FAILED,
         error_text);
+
+    uint32_t hash_id = packet->message_id();
```

Comment:
@jacobkeeler can be const?

Reply:
This value is changed in the case of an error in `OnSessionEndedCallback`, it needs to be mutable